src/test/java/api/support/matchers/LoanMatchers.java

```diff
@@ -54,9 +128,11 @@ public void describeTo(Description description) {
       }
 
       @Override
-      protected boolean matchesSafely(JsonObject representation, Description description) {
+      protected boolean matchesSafely(JsonObject representation,
+        Description description) {
         this.describeMismatch(representation.getValue("userId"), description);
-        return !representation.containsKey("userId") && !representation.containsKey("borrower");
+        return !representation.containsKey("userId") && !representation
+          .containsKey("borrower");
```

Comment:
Was this line split because it exceeds the intended maximum line length?

Reply:
yes.

Reply:
Personally I’d split it either just before or after the `&&` rather than between the object and the method.

Reply:
done